src/components/Tag/PTag.vue

```diff
@@ -23,12 +23,13 @@
   import { Icon } from '@/types/icon'
   import { TagValue, normalize } from '@/types/tag'
 
-  const props = defineProps<{
+  const props = withDefaults(defineProps<{
     icon?: Icon,
     label?: string,
     value?: string | TagValue,
     dismissible?: boolean,
-  }>()
+    size?: 'xs' | 'sm',
```

Comment:
To be a little more semantically prescriptive, maybe the terms here would be better suited as `'sm' | 'default'`? Or, like buttons, we ended up refactoring to just the default and a bool prop called `small`.

Reply:
I'm like 98.6% sure we'll never add a `large` for tags

Reply:
👍 will shoot for consistency. size -> `small?: boolean`